integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiRollingRestartLivenessProbe.java

```diff
@@ -0,0 +1,618 @@
+  private static final String miiImageName = "mdlb-mii-image";
+  private static final String wdtModelFileForMiiDomain = "model-multiclusterdomain-sampleapp-wls.yaml";
+  private static final String miiDomainUid = "mdlb-miidomain";
+
+  private static String opNamespace = null;
+  private static NginxParams nginxHelmParams = null;
+  private static int nodeportshttp = 0;
+  private static LoggingFacade logger = null;
+  private static String miiDomainNamespace = null;
+  private static String miiImage = null;
+
+  /**
+   * Install operator and NGINX.
+   * Create a model in image type of domain.
+   * Create ingress for the domain.
+   *
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   *                   JUnit engine parameter resolution mechanism
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
+    logger = getLogger();
+
+    // get a unique operator namespace
+    logger.info("Get a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique NGINX namespace
+    logger.info("Get a unique namespace for NGINX");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    String nginxNamespace = namespaces.get(1);
+
+    // get unique namespaces for three different type of domains
+    logger.info("Getting unique namespaces for three different type of domains");
+    assertNotNull(namespaces.get(2));
+    miiDomainNamespace = namespaces.get(2);
+
+    // set the service account name for the operator
+    String opServiceAccount = opNamespace + "-sa";
+
+    // create mii image
+    miiImage = createAndPushMiiImage();
+
+    // install and verify operator with REST API
+    installAndVerifyOperator(opNamespace, opServiceAccount, true, 0, miiDomainNamespace);
+
+    // This test uses the operator restAPI to scale the domain. To do this in OKD cluster,
+    // we need to expose the external service as route and set tls termination to  passthrough
+    logger.info("Create a route for the operator external service - only for OKD");
+    createRouteForOKD("external-weblogic-operator-svc", opNamespace);
+
+    // Patch the route just created to set tls termination to passthrough
+    setTlsTerminationForRoute("external-weblogic-operator-svc", opNamespace);
+
+    if (!OKD) {
+      // install and verify NGINX
+      nginxHelmParams = installAndVerifyNginx(nginxNamespace, 0, 0);
+      String nginxServiceName = nginxHelmParams.getHelmParams().getReleaseName() + "-ingress-nginx-controller";
+      logger.info("NGINX service name: {0}", nginxServiceName);
+      nodeportshttp = getServiceNodePort(nginxNamespace, nginxServiceName, "http");
+      logger.info("NGINX http node port: {0}", nodeportshttp);
+    }
+  }
+
+  /**
+   * Verify liveness probe by killing managed server process 3 times to kick pod container auto-restart.
+   */
+  @Test
```

Comment:
Can u try to merge the test method into ItLivenessProbeCustomization.java to aggregate all livenessProbe scenario.

Reply:
moved it to ItLivenessProbeCustomization.java